python/social_bot/envs/gazebo_base.py

```diff
@@ -134,6 +134,7 @@ def render(self, mode='human'):
     def close(self):
         super().close()
         gazebo.close_without_model_base_fini()
+        self.__del__()
```

Comment:
`__del__` is supposed to be called when the object is deleted. Strictly speaking, `close()` does not mean the object is deleted. So I suggest removing `__del__` and move the content here.